deployment/terraform/variables.tf

```diff
@@ -0,0 +1,221 @@
+variable "project" {
+  default = "Flood Map"
+  type    = string
+}
+
+variable "environment" {
+  default = "Staging"
+  type    = string
+}
+
+variable "aws_region" {
+  default = "us-east-1"
+  type    = string
+}
+
+variable "aws_availability_zones" {
+  default = ["us-east-1a", "us-east-1b"]
+  type    = list(string)
+}
+
+variable "aws_key_name" {
+  type = string
+}
+
+variable "r53_private_hosted_zone" {
+  type = string
+}
+
+variable "vpc_cidr_block" {
+  default = "10.0.0.0/16"
+  type    = string
+}
+
+variable "external_access_cidr_block" {
+  type = string
+}
+
+variable "vpc_private_subnet_cidr_blocks" {
+  default = ["10.0.1.0/24", "10.0.3.0/24"]
+  type    = list(string)
+}
+
+variable "vpc_public_subnet_cidr_blocks" {
+  default = ["10.0.0.0/24", "10.0.2.0/24"]
+  type    = list(string)
+}
+
+variable "bastion_ami" {
+  type = string
+}
+
+variable "bastion_instance_type" {
+  default = "t3.nano"
+  type    = string
+}
+
+variable "bastion_ebs_optimized" {
+  default = true
+  type    = bool
+}
+
+variable "rds_allocated_storage" {
+  default = 32
+  type    = number
+}
+
+variable "rds_engine_version" {
+  default = 11
+  type    = number
+}
+
+variable "rds_parameter_group_family" {
+  default = "postgres11"
+  type    = string
+}
+
+variable "rds_instance_type" {
+  default = "db.t3.micro"
+  type    = string
+}
+
+variable "rds_storage_type" {
+  default = "gp2"
+  type    = string
+}
+
+variable "rds_database_identifier" {
+  type = string
+}
+
+variable "rds_database_name" {
+  type = string
+}
+
+variable "rds_database_username" {
+  type = string
+}
+
+variable "rds_database_password" {
+  type = string
+}
+
+variable "rds_backup_retention_period" {
+  default = 30
+  type    = number
+}
+
+variable "rds_backup_window" {
+  default = "04:00-04:30"
+  type    = string
+}
+
+variable "rds_maintenance_window" {
+  default = "sun:04:30-sun:05:30"
+  type    = string
+}
+
+variable "rds_auto_minor_version_upgrade" {
+  default = true
+  type    = bool
+}
+
+variable "rds_final_snapshot_identifier" {
+  default = "fieldscope-rds-snapshot"
+  type    = string
+}
+
+variable "rds_monitoring_interval" {
+  default = 60
+  type    = number
+}
+
+variable "rds_skip_final_snapshot" {
+  default = false
+  type    = bool
+}
+
+variable "rds_copy_tags_to_snapshot" {
+  default = true
+  type    = bool
+}
+
+variable "rds_multi_az" {
+  default = false
+  type    = bool
+}
+
+variable "rds_storage_encrypted" {
+  default = false
+  type    = bool
+}
+
+variable "rds_deletion_protection" {
```

Comment:
This variable exists at the project level, but it isn't being set at the module level.

Reply:
Shoot, thanks for catching that. Fixed!